src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
 
+        truck.doWork();
+        bulldozer.doWork();
+        excavator.doWork();
+
+        truck.stopWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
```

Comment:
here better to create an array of Machines with Truck, Bulldozer and Excavator and call methods doWork and stopWork in a loop